src/registrar/management/commands/load_transition_domain.py

```diff
@@ -0,0 +1,463 @@
+    Updating entry: {existingEntry}
+    Status: {existingEntry.status} > {new_entry_status}
+    Email Sent: {existingEntry.email_sent} > {new_entry_emailSent}
+    {termColors.ENDC}"""
+                                )
+
+                            existingEntry.status = new_entry_status
+
+                        existingEntry.email_sent = new_entry_emailSent
+                        existingEntry.save()
+                    except TransitionDomain.DoesNotExist:
+                        # no matching entry, make one
+                        newEntry = TransitionDomain(
+                            username=new_entry_email,
+                            domain_name=new_entry_domainName,
+                            status=new_entry_status,
+                            email_sent=new_entry_emailSent,
+                        )
+                        to_create.append(newEntry)
+                        total_new_entries += 1
+
+                        # DEBUG:
+                        if debug_on:
+                            logger.info(
+                                f"{termColors.OKCYAN} Adding entry {total_new_entries}: {newEntry} {termColors.ENDC}"  # noqa
+                            )
+                    except TransitionDomain.MultipleObjectsReturned:
+                        logger.info(
+                            f"""
+{termColors.FAIL}
+!!! ERROR: duplicate entries exist in the
+transtion_domain table for domain:
+{new_entry_domainName}
+----------TERMINATING----------"""
+                        )
+                        import sys
+
+                        sys.exit()
+
+                # DEBUG:
+                if debug_on or debug_max_entries_to_parse > 0:
+                    if (
+                        total_rows_parsed > debug_max_entries_to_parse
+                        and debug_max_entries_to_parse != 0
+                    ):
+                        logger.info(
+                            f"""{termColors.WARNING}
+                            ----PARSE LIMIT REACHED.  HALTING PARSER.----
+                            {termColors.ENDC}
+                            """
+                        )
+                        break
+
+        TransitionDomain.objects.bulk_create(to_create)
+
+        logger.info(
+            f"""{termColors.OKGREEN}
+
+        ============= FINISHED ===============
+        Created {total_new_entries} transition domain entries,
+        updated {total_updated_domain_entries} transition domain entries
+        {termColors.ENDC}
+        """
+        )
+
+        # Print a summary of findings (duplicate entries,
+        # missing data..etc.)
+        self.print_summary_duplications(
+            duplicate_domain_user_combos, duplicate_domains, users_without_email
+        )
+        self.print_summary_status_findings(domains_without_status, outlier_statuses)
+
+    def print_debug_mode_statements(self, debug_on, debug_max_entries_to_parse):
```

Comment:
Ah, I'm definitely more in the camp of helper functions coming after the main function since it makes it super easy to find and read what the script is accomplishing, with subroutines appearing more like footnotes.  However, I can totally rearrange it and add those docstring comments.